src/app/components/applications/SSV/ValidatorRegistrationConfirmation/ValidatorRegistrationConfirmation.tsx

```diff
@@ -210,7 +214,7 @@ const ValidatorRegistrationConfirmation = () => {
               <NameAndAddress name={'SSV Deposit'} />
             </Grid>
             <Grid item>
-              <NameAndAddress name={`${'registerValidator' in process ? process.registerValidator?.depositAmount : 0} SSV`} />
+              <NameAndAddress name={`${totalAmountOfSsv} ssSSV`} />
```

Comment:
@Chris-ssvlabs I think this one is a typo "ssSSV" Change it back to "SSV"

Reply:
You are right, I missed that, Thank you :)